src/deploy/functions/runtimes/node/parseTriggers.ts

```diff
@@ -140,6 +140,9 @@ export function useStrategy(): Promise<boolean> {
   return Promise.resolve(true);
 }
 
+/**
+ * Parse trigger annotations in sourceDir to generate backed.Build.
```

Comment:
nit: `{@link backend.Build}`

+others

Reply:
We don't use api-documenter for CLI 🤔 Do you think that annotation is useful regardless?